src/components/InstrumentPicker/__tests__/InstrumentListItem.spec.tsx

```diff
@@ -0,0 +1,322 @@
+import {shallow} from 'enzyme';
+import * as React from 'react';
+import {AssetModel, InstrumentModel} from '../../../models';
+import {formattedNumberWithDashes} from '../../../utils/localFormatted/localFormatted';
+import {colors} from '../../styled';
+import InstrumentListItem from '../InstrumentListItem';
+
+describe('<InstrumentListItem>', () => {
+  let baseAsset: AssetModel;
+  let instrument: InstrumentModel;
+  let inactive: boolean;
+  let isAuth: boolean;
+  let onPick: (instrument: InstrumentModel) => void;
+
+  const getTestIntrumentListItem = () => {
+    return (
+      <InstrumentListItem
+        baseAsset={baseAsset}
+        instrument={instrument}
+        inactive={inactive}
+        isAuth={isAuth}
+        onPick={onPick}
+      />
+    );
+  };
+
+  beforeEach(() => {
+    baseAsset = new AssetModel({
+      id: 'BTC',
+      name: 'BTC',
+      canBeBase: true
+    });
+
+    instrument = new InstrumentModel({
+      id: 'BTCUSD',
+      baseAsset,
+      quoteAsset: new AssetModel({id: 'USD'}),
+      ask: 100
+    });
+
+    inactive = true;
+    isAuth = true;
+    onPick = jest.fn();
+  });
+
+  it('should render table row', () => {
```

Comment:
Please wrap blocks into describe('method render', () => {})
It will allow us to separate tests for rendered contents from tests for some functions and life-cycle methods